### DISCUSSION TITLE
Implement custom ListBox SelectionMode

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [AvaloniaUI/Avalonia#10276] [Q&A] [answered]
Hi, I would like to create a SelectionMode similar to `Multiple` but instead more like a toggle, i.e. you can click on a ListBoxItem to enable/disable but won't deselect every other selected item. I know this probably better served by another control but I would like to use ListBox for stylistic consistency.

**Comment by jp2masa** [comment, marked as answer]
I think you can use `SelectionMode` as flags, i.e. `SelectionMode.Multiple | SelectionMode.Toggle` in C#, and IIRC `Multiple, Toggle` in XAML.

**Reply by IsaacMarovitz** [reply]
Ahh, I didn't know that. It works perfectly
